### DISCUSSION TITLE
Tuning Media Block Size

### DISCUSSION THREAD

**Discussion by victorstewart** [hse-project/mpool#4] [Q&A]
Is there any material value in tuning the media block size for your given application based on the average and range of sizes of objects you'll be dealing with? 

I'll be using it through HSE only so maybe the size of my objects doesn't matter as much as the way HSE uses the memory?

**Comment by smoyerx** [comment]
We've tuned HSE for a 32MB mblock size, which is why we made 32MB the default for mblocks in mpool.  For the various workloads we've tested, there isn't anything to be gained by using a different mblock size.
